packages/date/src/element.ts

```diff
@@ -0,0 +1,113 @@
+import { html, LitElement, property, PropertyValues, TemplateResult } from 'lit-element';
+import { format, formatDistance, formatRelative, Locale } from 'date-fns';
+import { enGB } from 'date-fns/locale';
+
+const defaultDateFormat = 'dd.MM.yyyy';
+const defaultTimeFormat = 'HH:mm';
+const defaultTimeSecondsFormat = 'HH:mm:ss';
+const defaultDateTimeFormat = 'HH:mm dd.MM.yyyy';
+const defaultDateTimeSecondsFormat = 'HH:mm:ss dd.MM.yyyy';
+
+export type DateVariant = 'date' | 'time' | 'datetime' | 'relative' | 'distance' | 'custom';
+
+export type WeekDay = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface DateElementProps {
+  addSuffix?: boolean;
+  capitalizeFirstLetter?: boolean;
+  date?: Date;
+  baseDate?: Date;
+  format?: string;
+  includeSeconds?: boolean;
+  locale?: Locale;
+  variant?: DateVariant;
+  weekStartsOn?: WeekDay;
+}
+
+export class DateElement extends LitElement implements DateElementProps {
```

Comment:
there should be 2 elements

1.  A date element which render an `<time>` element which takes an `date` prop
2. A date span element which renders relative time between `from` and `to`

the React Component can be a single which makes some logic around date and relative date

Reply:
The <fwc-date> component has now been split into <fwc-datetime> and <fwc-daterange>